### ISSUE TITLE
[Question] Which file contains the error?

### ISSUE THREAD

**Issue by mimmi20** [stylelint/stylelint#9453]
I use Stylelint 17.14.1 and call it this way

```bash
npx stylelint "httpdocs/**/*.scss" --color --cache
```

I get errors like this:

```bash
<input css 1>
  78:35  ✖  Unknown word $css  CssSyntaxError

```

Is there a way for stylelint to tell me which file caused the error?

**Comment by jeddy3** [comment]
The default formatter output should include the file path when linting files:

```
% npx stylelint "code/test1.css"

code/test1.css
  2:12  ✖  Unknown value "1" for property "color"  declaration-property-value-no-unknown

✖ 1 problem (1 error, 0 warnings)
```

If you [follow the getting started guide](https://stylelint.io/user-guide/get-started#linting-css-like-languages-and-css-within-containers) and still get the same error, please [open a new issue using the bug template](https://github.com/stylelint/stylelint/issues/new?template=REPORT_A_BUG.yml) because it captures the information we'll need to help you.
